src/hearings/containers/request-hearing/hearing-view-edit-summary/hearing-view-edit-summary.component.spec.ts

```diff
@@ -10,6 +10,8 @@ import { HearingsService } from '../../../services/hearings.service';
 import { HearingViewEditSummaryComponent } from './hearing-view-edit-summary.component';
 import { Section } from '../../../../hearings/models/section';
 import { ScreenNavigationModel } from '../../../../hearings/models/screenNavigation.model';
+import * as fromHearingStore from '../../../../hearings/store';
+import { Store } from '@ngrx/store';
```

Comment:
external imports should be above internal imports

Reply:
Moved